kclvm/tools/src/LSP/src/goto_def.rs

```diff
@@ -200,30 +200,40 @@ pub(crate) fn resolve_var(
         _ => {
             let name = names[0].clone();
             match current_scope.lookup(&name) {
-                Some(obj) => {
-                    match &obj.borrow().ty.kind {
-                        kclvm_sema::ty::TypeKind::Schema(schema_type) => {
-                            find_attr_in_schema(schema_type, &node_names[1..], scope_map)
+                Some(obj) => match &obj.borrow().ty.kind {
+                    kclvm_sema::ty::TypeKind::Schema(schema_type) => {
+                        find_attr_in_schema(schema_type, &node_names[1..], scope_map)
+                    }
+                    kclvm_sema::ty::TypeKind::Module(module_ty) => {
+                        match scope_map.get(&pkgpath_without_prefix!(module_ty.pkgpath)) {
+                            Some(scope) => {
+                                return resolve_var(&node_names[1..], &scope.borrow(), scope_map);
+                            }
+                            None => None,
                         }
-                        kclvm_sema::ty::TypeKind::Module(module_ty) => {
-                            match scope_map.get(&pkgpath_without_prefix!(module_ty.pkgpath)) {
-                                Some(scope) => {
-                                    return resolve_var(
-                                        &node_names[1..],
-                                        &scope.borrow(),
-                                        scope_map,
-                                    );
+                    }
+                    kclvm_sema::ty::TypeKind::Dict(DictType { attrs, .. }) => {
+                        if !attrs.is_empty() {
+                            let start_pos = attrs.first().unwrap().1.range.0.clone();
```

Comment:
Why use `first()` instead of `attrs.get(&name)`? Besides, add a dict case containing two more elements.